### PR TITLE
addressed comments from PR #25

### PR DESCRIPTION
**Pull request by corpetty**
Addresses comments from PR #25

### CODE REVIEW

**Comment by corpetty** [comment]
new laptop, so not verified, will do tomorrow

File: x5.md [hunk 1]
```diff
@@ -90,19 +53,18 @@ This can be done in a the following ways:
...
   Signature               // Prekey signature
   Timestamp               // When the bundle was lasted created locally
-  XXX_NoUnkeyedLiteral    // ???
-  XXX_unrecognized        // ???
-  XXX_sizecache           // ???
   ```
   - include BundleContainer???
 - a new bundle SHOULD be created at least every 12 hours
+- a bundle is only generated when it is used
+- a bundle MUST be distributed on the contact code channel (NOTE: define this where?)
```
Comment (oskarth) [inline x5.md:66]: Can we leave comments in markdown comment format? To make sure the spec is readable as is

File: x5.md [hunk 2]
```diff
@@ -117,215 +79,45 @@ Status generates a deterministic 3-word random pseudonym from the whisper chat p
...
-#### Asynchronous (YES)
-> Trust establishment can occur asynchronously without all conversation participants online.
+possible connections
+- client - client (not really ever, this is facilitated through all other connections)
+    - personal chat
+        - ratcheted with X3DH
+    - private group chat
+        - pairwise ratcheted with X3DH
+    - public chat
+- client - mailserver (statusd + ???)
```
Comment (oskarth) [inline x5.md:94]: What's the action to get clarity on these unknowns? Same as rendezvous info below

**Review by oskarth — COMMENTED** [review]
Thanks for quick clean up, happy to merge and we can expand after

Left some comments in line

**Comment by oskarth** [comment]
Merging so it's easier for ongoing cleanup effort to build on this, if needed

**Pull request merged**